docs/team/green-tea-123.md

```diff
@@ -12,16 +12,50 @@ to companies for billing.
 
 Given below are my contributions to the project.
 
-* **New Feature**: NIL
-
-* **Code contributed**: NIL
+* **New Feature**: Added the ability to add companies into the address book.
+  * What it does: Allows the user to add representations of companies into the
+  address book.
+  * Justification: Jobs are tied to companies, and so recruiters inevitably
+  have to work with companies as well. Adding companies to the address book
+  allows recruiters to manage companies associated to their job listings and
+  contacts, and enhance their existing workflow.
+  * Highlights: This enhancement required the team to consider the predicate 
+  to a company being unique and how companies will interact with jobs and
+  contacts in the address book.
+* **New Feature**: Added the ability to list all companies in the address book.
+  * What it does: Allows the user to list all companies stored in the address book.
+  * Justification: Companies need to be easily shown to recruiters for them to
+  include companies and company-related features into their workflow.
+  * Highlights: The structure of the address book's GUI had to be debated when
+  the company list was added.
+* **New Feature**: Added the integration between companies and jobs.
+  * What it does: Tags jobs to a company when a job is created, and deletes
+  all jobs tagged to a company when the company is deleted.
+  * Justification: A job has to come from a company, so the constraint that
+  jobs must be tagged to a company already existing in the address book should
+  be followed. This facilitates other features that require the association 
+  between a job and a company to enhance a recruiter's workflow.
+  * Highlights: This enhancement had to be done in a non-conventional way due
+  to how the address book is saved, and was the subject of quite a few discussions
+  as to how to handle the integrations.
+
+* **Code contributed**: [RepoSense Link](https://nus-cs2103-ay2425s1.github.io/tp-dashboard/?search=green-tea-123&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2024-09-20&tabOpen=true&tabType=authorship&tabAuthor=Green-Tea-123&tabRepo=AY2425S1-CS2103-F13-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
 * **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+  * Managed release `v1.4` on GitHub
 
 * **Enhancements to existing features**: NIL
 
 * **Documentation**:
   * User Guide: NIL
-  * Developer Guide: NIL
+  * Developer Guide:
+    * Added use cases for company features [#45](https://github.com/AY2425S1-CS2103-F13-4/tp/pull/45)
+* **Community**:
+  * PRs reviewed (with non-trivial comments): 
```

Comment:
I am not sure if this is referring to the PRs done for our tP repo, since it is under the section `Community`, but I could not find any further information on it.

Reply:
those were PRs that I helped to review

Reply:
I understand, but those PRs are from our tP repo right? So I am not sure if it fit under the section `Community`

Reply:
oh so that's what you meant by community, I thought our team counted as community

Reply:
I think maybe we can have a discussion on this on Monday